nss-day-cohort-34/bangazon-site-the-naughty-list
# Orderhistory

# Description

Completed functionality for viewing and editing profile information

Related to Issue # 

21

# Testing Instructions

Click on your username in the top right corner and you should see pre-populated fields for name, phone number, and address. You should be able to edit these fields as well.

# Checklist:
